### ISSUE TITLE
Autoclose Timerpicker when hour (and/or minutes) are selected

### ISSUE THREAD

**Issue by DienH** [nj-coder/nj-timepicker#7]
It would be great if the timepicker could be automatically closed when hours and minutes are selected, without needing to click the **_save_** button.
It would be activated with an option like `autoHide: true`

I've managed to do so with a _onclick_ function on the '_div.nj-hours-container_' element that simulate a click on the **_save_** button if the current hour has changed, or if the current selected hour is clicked.

```
var hourPicker = new NJTimePicker({...})
hourPicker.hours.lastValue = hourPicker.hours.currentValue
hourPicker.hours.element.onclick = (event)=> {
 if (event.target.classList.contains('selected') || (hourPicker.hours.lastValue != hourPicker.hours.currentValue))
 {
  hourPicker.hours.lastValue = hourPicker.hours.currentValue
  hourPicker.buttons.element.querySelector('div.nj-action-button.nj-action-save').click()
 }
}
```

**Comment by nj-coder** [comment]
yep, will do.

**Comment by nj-coder** [comment]
added `autoSave` config, included in `v1.3.117`
